src/utils/block.ts

```diff
@@ -15,10 +15,28 @@ import { codec, Schema } from '@liskhq/lisk-codec';
 import { Database } from '@liskhq/lisk-db';
 import { BlockHeader } from '@liskhq/lisk-chain';
 
-import { DB_KEY_BLOCKS_ID } from '../constants';
+import { DB_KEY_BLOCKS_HEIGHT, DB_KEY_BLOCKS_ID } from '../constants';
 import { blockHeaderSchema } from '../schemas';
 import { keyString, incrementOne } from './transaction';
 
+export const formatInt = (num: number | bigint): string => {
```

Comment:
Cant we use `liskDB.formatInt(num)` directly from SDK

Reply:
I have verified this method is not exposed from `lisk-db` anymore

Reply:
ah ok, it was exposed via `lisk-sdk` we can ignore this.